From 9c3e385ca1afc7c321beaa8792c2fb64ee443f3f Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Wed, 28 Sep 2022 16:19:14 +0300
Subject: [PATCH] [SW-106171] gaudi3: tpm: remove redundant spi bus claim

This fix is a carry over from gaudi2. spi bus claim and release is to
be done as a part of spi_xfer function. And the outer redundant claim
and release can be removed.

This has no functional impact.

Change-Id: If2ade92e5cc0331d017a004957e68966853c1971
---

diff --git a/arch/gaudi3/arc/tpm2.c b/arch/gaudi3/arc/tpm2.c
index d10120b..32aa8de 100644
--- a/arch/gaudi3/arc/tpm2.c
+++ b/arch/gaudi3/arc/tpm2.c
@@ -84,8 +84,6 @@
 		return -1;
 	}
 
-	spi_tpm_claim_bus();
-
 	while (len) {
 		/* Request */
 		transfer_len = min_t(uint16_t, len, MAX_SPI_FRAMESIZE);
@@ -133,8 +131,6 @@
 		if (ret) {
 			goto release_bus;
 		}
-
-		spi_tpm_release_bus();
 
 		if (in) {
 			memcpy(in, rx_buf, transfer_len);
